radio/src/targets/horus/CMakeLists.txt

```diff
@@ -246,6 +246,11 @@ if (AUX2_SERIAL)
   set(AUX_SERIAL_DRIVER ../common/arm/stm32/aux_serial_driver.cpp)
 endif()
 
+if (PCB STREQUAL X10 AND PCBREV STREQUAL T16)
+  set(TARGET_SRC ${TARGET_SRC} mavlink_telem_driver.cpp)
+  set(SRC ${SRC} mavlink_telem.cpp mavlink_telem_autopilot.cpp mavlink_telem_camera.cpp mavlink_telem_gimbal.cpp)
```

Comment:
here you may add an option:
`option(TELEMETRY_MAVLINK "Mavlink Telemetry support" YES)`

and later:
```
if(TELEMETRY_MAVLINK)
  add_definition(-DTELEMETRY_MAVLINK)
endif()
```

Reply:
I think I would add the option above though, not sure

Reply:
I've changed things here. 
I'm not sure it's now exactly what you wanted. The option() thing was unclear to me.
It compiles now with specifying a CMAKE_FLAG=TELEMETRY_MAVLINK=YES, as well as with not specifying it, and I hope I got it all right.